src/flint/routers/api.py

```diff
@@ -34,7 +37,10 @@ async def chat(
     From: str = Form(...),
     Body: Optional[str] = Form(None),
     To: str = Form(...),
+    MediaUrl0: Optional[str] = Form(None),
+    MediaContentType0: Optional[str] = Form(None),
```

Comment:
Things to verify:
1. Can send multilingual messages
2. API still works if there's no media message included